gradle.properties

```diff
@@ -24,7 +24,7 @@ android.defaults.buildfeatures.buildconfig=true
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 
-project_version=1.2.5
+project_version=1.4.1-SNAPSHOT
```

Comment:
I am uncertain about which version to use in this context. Please feel free to make any necessary updates.